docs/learn/fundamentals/contract-development/storage/state-archival.mdx

```diff
@@ -21,7 +21,7 @@ Learn about choosing the right storage for your use case in this [How-To Guide](
 All contract data has a Time To Live (TTL) that must be periodically extended. If an entry's TTL is not periodically extended, the entry's TTL will eventually go to 0 and either become "archived" or permanently deleted depending on the storage type. Each type of storage functions similarly, but has different fees and archival behavior:
 
 - When a `Temporary` entry's TTL is 0, it is deleted from the ledger and is permanently inaccessible.
-- When a `Persistent` or `Instance` entry TTL is 0, it becomes inaccessible and is "archived", but can be "restored" and used again via the [`RestoreFootprintOp`](#restorefootprintop).
+- When a `Persistent` or `Instance` entry TTL is 0, it is "archived" but can be "restored" automatically via the `InvokeHostFunction` operation.
```

Comment:
I'd rephrase this a bit:
- When a `Persistent` or `Instance` entry TTL is 0, it is "archived" and can't be accessed until it is "restored". There are 2 ways to restore entries:
  - (usually) Automatically when an archived entry is accessed by `InvokeHostFunction` operation
  - (rarely) By manually building the [`RestoreFootprintOp`](#restorefootprintop)